src/vue/components/resource-settings.vue

```diff
@@ -34,21 +30,42 @@
     components: {
       setting: Setting,
     },
-    computed: {
-      resourceSettingsLexicons: function () {
+    data () {
+      return {
+        settingsArray: [
```

Comment:
As `typeLex` is a key maybe to add a comment about that so we won't forget that it must be unique?

Reply:
ok let's do it